Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```
@@ -163,6 +168,8 @@
   - type: FlashImmunity
   - type: EyeProtection
     protectionTime: 5
+  - type: Construction
+    graph: GlassesSecHUD
```

needs to specify the node for the graph.
```suggestion
    graph: GlassesSecHUD
    node: glassesSec
```

there's your fail